### DISCUSSION TITLE
Qtree - tickStrategy - strict, with indeterminate

### DISCUSSION THREAD

**Discussion by TomB30** [quasarframework/quasar#15693] [General - Components / Directives / etc]
Hello, I'm using the QTree component with tickStrategy: 'strict', And I need to mark the parents of the selected node.
I can't find a way to set the indeterminate of the parent checkbox with the 'strict' mode.
please provide support for indeterminate with 'strict' tickStrategy.

**Comment by ishaiboro** [comment]
Thanks @TomB30 I find it very useful too
